test/unit/project_type/test_git.py

```diff
@@ -66,11 +66,11 @@ def test_execute_command_in_project_type_specifies_cwd_if_doesnt_exist(self):
 
     def test_get_full_repo_directory(self):
```

Comment:
Want to add a test case for a Windows style path (since that is the situation this PR is fixing)? :wink: